src/pudl/constants.py

```diff
@@ -1410,7 +548,7 @@
     },
     'epacems': {
         'years': tuple(range(1995, 2021)),
-        'states': tuple(cems_states.keys())},
+        'states': tuple(EPACEMS_STATES.keys())},
     'ferc1': {
         'years': tuple(range(1994, 2020))
```

Comment:
`ferc714` provides all years of data in a single file so it's not possible to partition it by year (or anything else). Do you have any suggestions for how better to denote this?  Really I imagine these valid ranges and codes should be wrapped in a settings validation class of some kind.

Reply:
I find it more natural to name constants based on what is, not what can be. So instead of `WORKING_PARTITIONS`, this should probably be called e.g. `SOURCE_SUBSETS`. I would interpret `{}` as meaning "process nothing", so I would remove the 'ferc714' key (it is not accessed anywhere in the code). Absence then means "process everything; do not subset".

Reply:
We also use this information for validation -- if someone asks for the eia860 data for 2020, it'll fail, because it's not available. Really we need to be able to do the same thing for `ferc714` since the 2020 data isn't available there either.  This is also empty at the moment because we haven't fully integrated `ferc714` (pending the harvesting refactor) and putting an empty dictionary made it "do the right thing" for the moment which is process everything and never complain about it no matter what you ask for. But we'll need the valid range information stored somewhere when it is fully integrated.